test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareTestUtils.cs

```diff
@@ -61,12 +63,17 @@ internal static string CreateDacpac(SqlTestDb testdb)
             return extractParams.PackageFilePath;
         }
 
+        /// <summary>
+        /// Creates an SDK-style .sqlproj from the database
+        /// </summary>
+        /// <param name="testdb">Database to create the sql project from</param>
+        /// <param name="projectName">Name of the project</param>
+        /// <returns>Full path to the project folder</returns>
         internal static string CreateProject(SqlTestDb testdb, string projectName)
         {
             var result = GetLiveAutoCompleteTestObjects();
-            string folderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "SchemaCompareTest", projectName);
-            Directory.CreateDirectory(folderPath);
-            File.Create(Path.Combine(folderPath, projectName + ".sqlproj")).Close();
```

Comment:
this was previously creating an empty file with no contents that had the file extension `.sqlproj`. The actual .sqlproj isn't used in the schema compare, but I updated this to actually create a .sqlproj with the expected xml so it isn't confusing in the future.

Reply:
Constructing these sorts of tests should be much easier with the XML stuff I'm working on!